src/docs/testing/ui-performance/index.md

```diff
@@ -275,10 +265,13 @@ a menu for enabling or disabling the overlay.
 
 If the performance overlay shows red in the UI graph, start by profiling
 the Dart VM, even if the GPU graph also shows red. To do this, use
-[Observatory](https://dart-lang.github.io/observatory/), Dart’s profiling tool.
+[Observatory][], Dart’s profiling tool.
 
 #### Displaying Observatory
 
+TBD: What to say about the Flutter timeline view? Do we still support that? The
```

Comment:
No. We are creating a devtools timeline view that is more user friendly. https://github.com/flutter/devtools
tells you how to install and run it. 
you will soon be able to launch it more simply from intellij and vscode.